content/chat/moderation.textile

```diff
@@ -1,7 +1,38 @@
 ---
 title: Moderation
-meta_description: "Limit chat participant’s ability to post messages within a given timeframe within specific conversations, or entire conversation namespaces."
+meta_description: "Limit chat participant's ability to post messages within a given timeframe within specific conversations, or entire conversation namespaces."
 product: chat
 languages:
   - javascript
 ---
+
+As with any chat applications, there needs to be some functionality implemented to allow for moderation to ensure the chat participants don't breach the terms of use or carry out actions such as spamming conversations.
+
+h2(#slow-mode). Slow mode
+
```

Comment:
I still think that slow mode alone doesn't help much with moderating chat.

Reply:
https://github.com/ably/docs/pull/2090/commits/76c189c5d30b62e96e3a7e7051ed455562fe78ce

I've renamed it to Spam Protection.